keyboards/nullbitsco/snap/keymaps/typehud/README.md

```diff
@@ -0,0 +1,51 @@
+# Typehud Keymap
+
+VIA compatible keymap that displays a live wpm HUD on your OLED.
+
+<https://user-images.githubusercontent.com/2514771/166799498-01af70cd-fee3-40e4-861d-b3646f40051b.mp4>
```

Comment:
Please rename the file to be all lower case.

Also, ideally, this should be rehosted on a dedicated hosting site. converting to a gif and using Imgur should work fine.